resources/setting_visibility/expert.cfg

```diff
@@ -384,6 +384,12 @@ roofing_angles
 infill_enable_travel_optimization
 material_flow_dependent_temperature
 material_flow_temp_graph
+interlocking_enable
```

Comment:
@MariMakes and @casperlamboo shouldn't this be placed on level down in advanced

Reply:
Ah you're right! It was already there so it was already showing up correctly in the front end. 
https://github.com/Ultimaker/Cura/blob/540fca2409520d133c35ba49199d4092be406d0c/resources/setting_visibility/advanced.cfg#L142
But it shouldn't be duplicated. Will change this directly in main

Reply:
Correction, it should be there. If I remove it it will only show up in "advanced" and not in "expert"